x/amm/types/codec.go

```diff
@@ -12,25 +12,19 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgJoinPool{}, "amm/JoinPool", nil)
 	cdc.RegisterConcrete(&MsgExitPool{}, "amm/ExitPool", nil)
 	cdc.RegisterConcrete(&MsgSwapExactAmountIn{}, "amm/SwapExactAmountIn", nil)
+	cdc.RegisterConcrete(&MsgFeedMultipleExternalLiquidity{}, "amm/FeedMultipleExternalLiquidity", nil)
 	cdc.RegisterConcrete(&MsgSwapExactAmountOut{}, "amm/SwapExactAmountOut", nil)
 	// this line is used by starport scaffolding # 2
 }
 
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgCreatePool{},
-	)
-	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgJoinPool{},
-	)
-	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgExitPool{},
-	)
-	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgSwapExactAmountIn{},
-	)
-	registry.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
@jelysn what did we remove those registered implementations?

Reply:
Nothing's removed - just simplified the multiple registercall into single call with multiple messages

Reply:
gotcha thanks